doc/api/perf_hooks.md

```diff
@@ -808,7 +808,7 @@ performance.mark('test');
 performance.mark('meow');
 ```
 
-## `perf_hooks.createHistogram([options])`
+## `perf_hooks.createHistogram()`
```

Comment:
Should `[options]` be removed here? Based on my understanding, this PR _adds_ support for `options`.

Reply:
Doh, good catch. I had started to make an edit removing the options then decided in mid go that they really should be implemented.